pkgs/development/python-modules/flask-mysqldb/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, flask
+, mysqlclient
+}:
+
+buildPythonPackage rec {
+  pname = "flask-mysqldb";
+  version = "1.0.1";
+  format = "setuptools";
```

Upstream says [not explicitly](https://github.com/alexferl/flask-mysqldb/blob/master/setup.py#L35C1-L35C49) that the module is not working on Python < 3.7. Still, let's stay away from unsupported Python releases for new packages.

```suggestion
  format = "setuptools";

  disabled = pythonOlder "3.7";
```

---

Fixed.